code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -713,6 +731,8 @@
 	desc = "A katana of acceptable quality, mass-produced by Lonestar. Easily their best-selling product from this lineup."
 	icon_state = "katana_old"
 	item_state = "katana"
+	force = WEAPON_FORCE_DANGEROUS
```

Comment:
: /

Reply:
What's wrong with the cheap katana having less damage but more penetration?